make publish fails on import error

In a fresh tmp virtual env and older environments trying to use the new publisconf.py causes the following error

```
Traceback (most recent call last):
  File "/Users/bunnyman/Virtualenvs/6471e5872a4d526e/bin/pelican", line 8, in <module>
    load_entry_point('pelican==3.0', 'console_scripts', 'pelican')()
  File "/Users/bunnyman/Virtualenvs/6471e5872a4d526e/src/package/pelican/__init__.py", line 237, in main
    pelican = get_instance(args)
  File "/Users/bunnyman/Virtualenvs/6471e5872a4d526e/src/package/pelican/__init__.py", line 220, in get_instance
    settings = read_settings(args.settings)
  File "/Users/bunnyman/Virtualenvs/6471e5872a4d526e/src/package/pelican/settings.py", line 79, in read_settings
    local_settings = get_settings_from_file(filename)
  File "/Users/bunnyman/Virtualenvs/6471e5872a4d526e/src/package/pelican/settings.py", line 94, in get_settings_from_file
    execfile(filename, tempdict)
  File "/Users/bunnyman/Virtualenvs/6471e5872a4d526e/test/publishconf.py", line 4, in <module>
    from pelicanconf import *
ImportError: No module named pelicanconf
make: *** [publish] Error 1
```

--- a/pelican/tools/templates/publishconf.py.in
+++ b/pelican/tools/templates/publishconf.py.in
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*- #
 
+import sys
+sys.path.append('.')
 from pelicanconf import *
 
 SITEURL = '$siteurl'

